dingdongg/spotify-payment
# finished the routing for member home

Description

- Finished routing for member home

To Test

- npm run dev
- go to member home
- click on each of the profile
- make sure it goes to each of the designated pages
